### PR TITLE
Refactor NFR file structure

### PR DESCRIPTION
**Pull request by aleronupe**
Refactor NFR file with introduction, references and static images of frameworks

## Issue #
#17 

## Description
This issue add the modelling of NFR frameworks of security, support, usability and Privacy

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation

## Checklist:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

**Review by victorhdcoelho — DISMISSED** [review]
The frame of some diagrams need my google drive's access to load. Because of that a need permission to that document on drive for get view of it. I advise you to extract image of that diagrams and upload at the git.

File: docs/requisitos/modelagem/nfr.md [hunk 1]
```diff
@@ -1,26 +1,44 @@
...
+Modelo que analisa as relações de segurança e de privacidade do projeto.
 
-## 2. Segurança e Privacidade
+### 2.1. Modelo Estático
+
+![seguranca_e_privacidade](../../assets/requisitos/modelagem/nfr_seguranca_e_privacidade.jpg)
+
+### 2.2. Modelo Dinâmico
 
 <iframe frameborder="0" style="width:160%;height:400px;" src="https://www.draw.io/?lightbox=1&highlight=0000ff&edit=_blank&layers=1&nav=1#G1MJrp5h0Ak-EyIoB_GbuJ2dNzD2Rnbodu"></iframe>
```
Comment (gabibguedes) [inline docs/requisitos/modelagem/nfr.md:25]: I can't see this image with drawio, it seems like I don't have permission. Try change the permission so it's public or download it to the repo.

File: docs/requisitos/modelagem/nfr.md [hunk 2]
```diff
@@ -1,26 +1,44 @@
...
+Modelo que analisa as relações de suporte e usabilidade do projeto.
+
+### 3.1. Modelo Estático
+
+![suporte_e_usabilidade](../../assets/requisitos/modelagem/nfr_suporte_e_usabilidade.jpg)
+
+
+### 3.2. Modelo Dinâmico
+
 <iframe frameborder="0" style="width:200%;height:400px;" src="https://www.draw.io/?lightbox=1&highlight=0000ff&edit=_blank&layers=1&nav=1#G160iv-Dl_g6iKoA6OzDitaviQzbkm-6mR"></iframe>
```
Comment (gabibguedes) [inline docs/requisitos/modelagem/nfr.md:38]: Can't see this image either.

**Review by gabibguedes — CHANGES_REQUESTED** [review]

**Review by renan601 — APPROVED** [review]

**Review by gabibguedes — APPROVED** [review]
Awesome

**Pull request merged**